src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -279,6 +279,123 @@ public sealed class SocketsHttpHandler_HttpClientHandler_ServerCertificates_Test
         public SocketsHttpHandler_HttpClientHandler_ServerCertificates_Test(ITestOutputHelper output) : base(output) { }
     }
 
+    public sealed class SocketsHttpHandler_KeepAlivePing : HttpClientHandlerTestBase
+    {
+        public SocketsHttpHandler_KeepAlivePing(ITestOutputHelper output) : base(output) { }
+
+        protected override Version UseVersion => HttpVersion.Version20;
+
+        [Fact]
+        public void DefaultValues()
+        {
+            using (var handler = new SocketsHttpHandler())
+            {
+                Assert.Equal(TimeSpan.FromSeconds(20), handler.KeepAlivePingTimeout);
+                Assert.Equal(TimeSpan.FromSeconds(0), handler.KeepAlivePingDelay);
+                Assert.Equal(HttpKeepAlivePingPolicy.Always, handler.KeepAlivePingPolicy);
+            }
```

Comment:
Prefer a using declaration when the scope is obvious, to avoid excessive indentation.

```suggestion
            using var handler = new SocketsHttpHandler();
            Assert.Equal(TimeSpan.FromSeconds(20), handler.KeepAlivePingTimeout);
            Assert.Equal(TimeSpan.FromSeconds(0), handler.KeepAlivePingDelay);
            Assert.Equal(HttpKeepAlivePingPolicy.Always, handler.KeepAlivePingPolicy);
```